index.html

```diff
@@ -1,12 +1,732 @@
+                    <a href="#" class="nav__link"><i class="fab fa-pinterest"></i></a>
+                  </li>
+                  <li class="nav__item">
+                    <a href="#" class="nav__link"><i class="fab fa-facebook-f"></i></a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </article>
+      <article class="post">
+        <div class="post__thumbnail">
+          <a href="#"><img src="./images/Art-image7.png" alt="health" class="post__image"></a>
+        </div>
+        <div class="post__description">
+          <div class="post__header">
+            <div class="post__links">
+              <a class="post__links-item" href="#">ART</a>
+              <a class="post__links-item" href="#">HEALTH</a>
+            </div>
+            <h3 class="post__title">How To Meet That Special Someone</h3>
+            <div class="post__time">
+              <p class="post__date">June 14, 2015</p>
+              <a href="#" class="post__author">by Author Doe</a>
+            </div>
+          </div>
+          <div class="post__content">
+            <p class="post__text">Lorem Ipsum has been the industry's standard dummy text ever since the 1500s when an unknown printer took ...</p>
+            <a href="#" class="post__more">...</a>
+          </div>
+          <div class="post__footer">
+            <div class="post__icon">
+              <div class="post__icon-left">
+                <div class="post__comment">
+                  <a href="#" class="post__comment-link"><i class="far fa-comment"></i> 24</a>
+                </div>
+              </div>
+              <div class="post__icon-right">
+                <ul class="nav">
+                  <li class="nav__item">
+                    <a href="#" class="nav__link">Share</a>
+                  </li>
+                  <li class="nav__item">
+                    <a href="#" class="nav__link"><i class="fab fa-twitter"></i></a>
+                  </li>
+                  <li class="nav__item">
+                    <a href="#" class="nav__link"><i class="fab fa-pinterest"></i></a>
+                  </li>
+                  <li class="nav__item">
+                    <a href="#" class="nav__link"><i class="fab fa-facebook-f"></i></a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </article>
+    </div>
+    <!-- end-main-content -->
+    <!-- sidebar -->
+    <aside class="sidebar">
+      <section class="about-me">
+        <img src="./images/avatar-about-me.png" alt="about-me" class="about-me__avatar">
+        <h2 class="about-me__title">about me</h2>
+        <p class="about-me__des">Hi! Dummy text of the printing and typesetting industry. Lorem ipsum has been the industry's standard dummy text ever</p>
+        <p class="about-me__follow">follow me on</p>
+        <div class="about-me__social">
```

Comment:
Đổi lại class cho toàn bộ link của social

Reply:
em đang để các link của social list chung 1 class là social__link để em css chung cho nó có được không ạ